addon_preferences.py

```diff
@@ -9,6 +9,7 @@
 class ProxyPreferences(bpy.types.AddonPreferences):
     bl_idname = __package__
     video_export_path = StringProperty(
+        subtype="DIR_PATH",
```

Comment:
It's only this change that we're talking about, right? If so, you should have a single commit with that change in the PR. Message me anytime if you need help to either clean up this branch with git, or create a clean new PR. The first time it can be hard to figure it out alone as git provides so many features and options. You can use `git rebase -i` (interactive tool to remove, edit, merge... commits together) in this case to drop all unwanted commits on this branch.

Reply:
Hi, thanks for the input, yep understood, it was quite a mess the way i did it, not used to github at all ;) i deleted the branch and will re submit a clean pull request, sorrey about that ! 
cheers !